### ISSUE TITLE
Is this going to be supported over webpack-serve?

### ISSUE THREAD

**Issue by bw** [webpack/webpack-dev-server#1463]
* Operating System: Docker (node)
* Node Version: v10.9.0
* NPM Version: v6.2.0
* webpack Version: v4.17.0
* webpack-dev-server Version: N/A

- [ ] This is a **bug**
- [x] This is a **modification** request

I just spent the entire day migrating to webpack-serve from webpack-dev-server after the deprecation notice that was previously on the README. I check back this morning and it appears that the notice is removed?

Which is going to be supported moving forward, and which is the recommendation?

**Comment by michael-ciniawsky** [comment]
I can only advise you to hold off on that effort for now, we are in the process to support only **one solution** in the future to avoid issue like this. The original author of `webpack-serve` is no longer actively working on it and having two 'official'  solutions wasn't a good idea anyways. The next major version of `webpack-dev-server` might be based upon parts of `webpack-serve`, but that's all I can say about it atm, since we just started detangeling the mess.

**Comment by bw** [comment]
OK -- thanks for the clarification. `webpack-serve` wasn't working anyways (I couldn't get React hot reloading to work) so we've reverted back to `webpack-dev-server`.

**Comment by michael-ciniawsky** [comment]
Yes I will try to keep everyone updated, for now 

- [ROADMAP](https://github.com/webpack/webpack-dev-server/commit/418493d040a5200a709344427cf923baea036f1e)
- [Milestone](https://github.com/webpack/webpack-dev-server/milestone/4)

**Comment by ferdinando-ferreira** [comment]
>https://github.com/webpack-contrib/webpack-serve/issues/262#issuecomment-417328623
>@evilebottnawi commented
>Don't use webpack-serve, it will be PoC and all features will be shipped in webpack-dev-server in near future. In near future webpack-serve will be archived.

@michael-ciniawsky: is that a final decision? Other than a few warts `webpack-serve` (at least for my use case) is faster, more modern and it's codebase a whole lot more organized than `webpack-dev-server`.

**Comment by evilebottnawi** [comment]
@ferdinando-ferreira yes, `webpack-dev-server` will be migrate on koa and websockets in near future

**Comment by benwiley4000** [comment]
Kind of glad to hear this. I have been using webpack-serve for one of my projects and it generally works great with little config - so it's an awesome tool - but it's frustrating how the maintainer of that project has tended to respond to almost anyone who opens an issue with such a hostile/self-important tone, always rushing to assure folks it's probably their own fault.

**Comment by teh0xqb** [comment]
`webpack-serve` was officially deprecated and archived 6 days ago, commit:
https://github.com/webpack-contrib/webpack-serve/commit/edaf28c6c935c3e29d26dbb02a9d6b672cc5148a

There's also a discussion to update README in #1493
